src/cc65.props

```diff
@@ -0,0 +1,62 @@
+<Project>
+
+	<!-- VS2017 compatibility. -->
+	<PropertyGroup Condition="$(MSBuildVersion.Split('.')[0]) == '15'">
+		<PlatformToolset Condition="'$(PlatformToolset)' == ''">v141</PlatformToolset>
+		<WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">$([Microsoft.Build.Utilities.ToolLocationHelper]::GetLatestSDKTargetPlatformVersion('Windows', '10.0'))</WindowsTargetPlatformVersion>
+	</PropertyGroup>
+
+	<!-- Common settings. -->
+	<PropertyGroup>
+		<PlatformToolset Condition="'$(PlatformToolset)' == ''">v142</PlatformToolset>
+		<WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">10.0</WindowsTargetPlatformVersion>
+		<IntDir>$(SolutionDir)..\wrk\$(MSBuildProjectName)\$(Configuration)\</IntDir>
+		<OutDir>$(SolutionDir)..\bin\</OutDir>
+		<OutDir Condition="$(MSBuildProjectName) == 'common'">$(IntDir)</OutDir>
+	</PropertyGroup>
+	<ItemDefinitionGroup>
+		<ClCompile>
+			<PrecompiledHeader></PrecompiledHeader>
+			<WarningLevel>Level3</WarningLevel>
+			<TreatWarningAsError>true</TreatWarningAsError>
+			<PreprocessorDefinitions>_CRT_NONSTDC_NO_WARNINGS;_CRT_SECURE_NO_WARNINGS;_WIN32_WINNT=0x0601;WINVER=0x0601;NTDDI_VERSION=0x06010000;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+			<AdditionalIncludeDirectories Condition="$(MSBuildProjectName) != 'common'">common</AdditionalIncludeDirectories>
+		</ClCompile>
+		<Link>
+			<AdditionalDependencies Condition="$(MSBuildProjectName) != 'common'">$(IntDir)..\..\common\$(Configuration)\common.lib</AdditionalDependencies>
+		</Link>
+	</ItemDefinitionGroup>
+
+	<!-- Debug settings. -->
+	<PropertyGroup Condition="'$(Configuration)'=='Debug'">
+		<LinkIncremental>true</LinkIncremental>
+	</PropertyGroup>
+	<ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
+		<ClCompile>
+			<RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
+			<BufferSecurityCheck>true</BufferSecurityCheck>
+		</ClCompile>
+		<Link>
+			<GenerateDebugInformation>true</GenerateDebugInformation>
+		</Link>
+	</ItemDefinitionGroup>
+
+	<!-- Release settings. -->
+	<PropertyGroup Condition="'$(Configuration)'=='Release'">
+		<LinkIncremental>false</LinkIncremental>
+		<WholeProgramOptimization>true</WholeProgramOptimization>
+	</PropertyGroup>
+	<ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
+		<ClCompile>
+			<FunctionLevelLinking>true</FunctionLevelLinking>
+			<RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+			<BufferSecurityCheck>false</BufferSecurityCheck>
+		</ClCompile>
+		<Link>
+			<GenerateDebugInformation>false</GenerateDebugInformation>
+			<EnableCOMDATFolding>true</EnableCOMDATFolding>
+			<OptimizeReferences>true</OptimizeReferences>
+			<LinkTimeCodeGeneration>UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
+		</Link>
+	</ItemDefinitionGroup>
+</Project>
```

Comment:
Seems like you have unintentionally removed this final newline.

Reply:
@oliverschmidt seems like VS2019 removes them by default when you save a project file or a props file from GUI.

Reply:
I see :-(